manifests/params.pp

```diff
@@ -164,8 +164,8 @@
     'batch-size'                 => '200',
     'broker-list'                => '',
     'compression-codec'          => '',
-    'key-serializer'             => 'kafka.serializer.StringEncoder',
-    'line-reader'                => 'kafka.tools.ConsoleProducer$LineMessageReader',
+    'key-serializer'             => 'kafka.serializer.DefaultEncoder',
+    'line-reader'                => 'kafka.tools.ConsoleProducer\$LineMessageReader',
```

Comment:
Yes, this hash is passed through an .erb template. Otherwise the ```$LineMessageReader``` would be replaced with an whitespace.